src/Consolonia.Themes/Templates/Controls/Helpers/ConsoleScrollContentPresenter.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Reflection;
+using Avalonia;
+using Avalonia.Controls.Presenters;
+using Avalonia.Input;
+using Avalonia.Media;
+using Avalonia.Utilities;
+
+namespace Consolonia.Themes.Templates.Controls.Helpers
+{
+    // This is workaround for Avalonia issue https://github.com/AvaloniaUI/AvaloniaEdit/issues/540
```

Comment:
How to reproduce this issue? When I scroll with the wheel in edit.net branch it seems all good.

Reply:
Change the curses velocity to one
```   const double velocity = 1;```